starter-code/src/clue.js

```diff
@@ -92,3 +91,6 @@ var roomsArray = [];
 
 // Weapons Collection
 var weaponsArray = [];
+
+
+var mysteryCharacters = Math.floor(Math.random * charactersArray.length);
```

Comment:
```
Math.floor(Math.random * (charactersArray.length + 1))
```
but charactersArray should be pushed with all characters objects.

Reply:
Oops! Bad correction! If you are using the number as an index it's okay :disappointed: sorry